recipes/onnx/all/conanfile.py

```diff
@@ -62,7 +55,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires(f"protobuf/{self._protobuf_version}", run=not cross_building(self), transitive_headers=True, transitive_libs=True)
+        self.requires("protobuf/3.21.12", run=not cross_building(self), transitive_headers=True, transitive_libs=True)
```

Comment:
what is the reason of updating `protobuf` version? 
All other packages who uses ONNX and protobuf become broken and have to update protobuf as well:
```
Resolved version ranges
    ittapi/[>=3.23.0]: ittapi/3.24.0
    onetbb/[>=2021.2.1]: onetbb/2021.9.0
    pugixml/[>=1.10]: pugixml/1.13
    snappy/[>=1.1.7]: snappy/1.1.10
    xbyak/[>=6.62]: xbyak/6.73
ERROR: Version conflict: Conflict between protobuf/3.21.12 and protobuf/3.21.9 in the graph.
Conflict originates from onnx/1.13.1
```
See https://github.com/conan-io/conan/issues/14007#issuecomment-1701580924